includes/misc-functions.php

```diff
@@ -1997,17 +2011,17 @@ function give_goal_progress_stats( $form ) {
 	// Define Actual Goal based on the goal format.
 	switch ( $goal_format ) {
 		case 'percentage':
-			$actual     = "{$progress}%";
+			$actual     = $usePlaceholder ? $placeholder : "{$progress}%";
 			$total_goal = '';
 			break;
 
 		case 'amount' === $goal_format:
-			$actual     = give_currency_filter( give_format_amount( $actual ) );
+			$actual     = $usePlaceholder ? $placeholder : give_currency_filter( give_format_amount( $actual ) );
 			$total_goal = give_currency_filter( give_format_amount( $total_goal ) );
 			break;
 
 		default:
-			$actual     = give_format_amount( $actual, [ 'decimal' => false ] );
+			$actual     = $usePlaceholder ? $placeholder : give_format_amount( $actual, [ 'decimal' => false ] );
```

Comment:
It looks like all these conditionals could be simplified down the chain so they can remain the original way:

```php
$stats_array = array_merge(
		[
			'progress' => $progress,
			// here is the placeholder conditional
			'actual'   => $usePlaceholder ? $placeholder : $actual,
			'goal'     => $total_goal,
			'format'   => $goal_format,
		],
		$stats_array
	);
```

Reply:
Resolved in 7d8059274d5a01e5df31058fd2a1fec04ad245d2